test/upload-download.test.js

```diff
@@ -91,6 +91,7 @@ describe('storage service', function() {
       .set('Content-Type', 'application/json')
       .expect('Content-Type', /json/)
       .expect(200, function(err, res) {
+        if (err) done(err);
```

Comment:
Depending on the framework, test might stop at `done(err)`, but I'd say clear and objective code will be more easily debuggable in the future, so I'd suggest:
```
if (err) {
  return done(err);
}
```